declarations/Match.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Match",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://uk.match.com/pages/misc/cge",
```

Comment:
Wrong fetch url, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/d976b544c3e87c3626986f094dda0ddde795217e